WordPress/Classes/ViewRelated/Post/EditImageDetailsViewController.m

```diff
@@ -0,0 +1,552 @@
+#import "EditImageDetailsViewController.h"
+#import <QuartzCore/QuartzCore.h>
+#import <WordPress-iOS-Editor/WPImageMeta.h>
+#import <WordPress-iOS-Shared/UIImage+Util.h>
+#import <WordPress-iOS-Shared/UITableViewTextFieldCell.h>
+#import "AbstractPost.h"
+#import "PostSettingsSelectionViewController.h"
+#import "UIImageView+AFNetworkingExtra.h"
+#import "WPTableViewController.h"
+#import "WPTableViewSectionHeaderView.h"
+
+static NSString *const TextFieldCell = @"TextFieldCell";
+static NSString *const CellIdentifier = @"CellIdentifier";
+static NSString *const ThumbCellIdentifier = @"ThumbCellIdentifier";
+static CGFloat CellHeight = 44.0f;
+
+typedef enum {
```

Comment:
I'll only mention it once... NS_ENUM instead? Not sure if we landed on something official for the style guide. Besides you code style is so good that I have to find something to nitpick :ok_hand: